diffulex/utils/loader.py

```
@@ -44,6 +45,133 @@ def default_weight_loader(param: nn.Parameter, loaded_weight: torch.Tensor):
+                weight_loader(param, loaded_weight)
+            except (AttributeError, KeyError):
+                try:
+                    buffer = model.get_buffer(param_name)
+                    buffer.copy_(loaded_weight)
+                except (AttributeError, KeyError):
+                    pass
+        else:
+            try:
+                param = model.get_parameter(param_name)
+                weight_loader = partial(
+                    getattr(param, "weight_loader"),
```

Ruff (0.15.9)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)